src/lib/utils/isFiniteNumber.ts

```diff
@@ -0,0 +1,5 @@
+function isFiniteNumber(value: any): value is number {
```

Comment:
We can add a comment to the polyfill version, since we don't provide a test.

> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite#Polyfill